starter-code/app.js

```diff
@@ -1,5 +1,46 @@
+'use strict';
+
 const express = require('express');
 const app = express();
 const Chuck  = require('chucknorris-io');
 const client = new Chuck();
+const expressLayouts = require('express-ejs-layouts');
+
+//configure app
+app.set('views', __dirname + '/views');
+app.set('view engine', 'ejs');
+
+app.use(expressLayouts);
+app.use(express.static('static'));
+
```

Comment:
It is a very good idea to have a route for /, if you want to display the same thing that in /random you can do:
```
app.get("/", (req, res) => {
  res.redirect('/random');
});
```